kapacitor/content.md

```diff
@@ -1,45 +1,112 @@
 # Kapacitor
 
-Kapacitor is a data processing engine. It can process both stream and batch data.
+Kapacitor is an open source data processing engine written in Go. It can process both stream and batch data.
+
+[Kapacitor Official Docs](https://docs.influxdata.com/kapacitor/latest/introduction/getting_started/)
 
 %%LOGO%%
 
 ## Using this image
 
+
+### Using the default configuration
+
 Start the Kapacitor container with default options:
 
 ```console
-$ docker run --net=host \
-      -v /path/on/host/kapacitorFiles/:/var/lib/kapacitor
-      kapacitor
+$ docker run -p 9092:9092 \
+    -v $PWD:/var/lib/kapacitor \
+    kapacitor
+```
+
+Modify `$PWD` to the directory where you want to store data associated with the Kapacitor container.
+
+You can also have Docker control the volume mountpoint by using a named volume.
+
+```console
+docker run -p 9092:9092 \
+    -v kapacitor:/var/lib/kapacitor \
+    kapacitor
 ```
 
-Start the Kapacitor container with custom configuration.
+### Configuration
+
+Kapacitor can be either configured from a config file or using environment variables. To mount a configuration file and use it with the server, you can use this command:
+
+Generate the default configuration file:
+
+```console
+$ docker run --rm kapacitor kapacitord config > kapacitor.conf
+```
+
+Modify the default configuration, which will now be available under `$PWD`. Then start the Kapacitor container.
 
 ```console
-$ docker run --net=host \
-      -v /path/on/host/kapacitor.config:/etc/kapacitor/kapacitor.conf:ro \
-      -v /path/on/host/kapacitorFiles/:/var/lib/kapacitor
-      kapacitor
+$ docker run -p 9092:9092 \
+      -v $PWD:/etc/kapacitor:ro \
+      kapacitord -config /etc/kapacitor/kapacitor.conf
 ```
 
-Start the Kapacitor container with custom configuration using env vars.
+Modify `$PWD` to the directory where you want to store the configuration file.
+
+For environment variables, the format is `KAPACITOR_$SECTION_$NAME`. All dashes (`-`) are replaced with underscores (`_`). If the variable isn't in a section, then omit that part.
+
+Examples:
 
 ```console
-$ docker run --net=host
-      -e KAPACITOR_LOGGING_LEVEL=DEBUG \
-      -v /path/on/host/kapacitorFiles/:/var/lib/kapacitor
-      kapacitor
+KAPACITOR_HOSTNAME=kapacitor
+KAPACITOR_LOGGING_LEVEL=INFO
+KAPACITOR_REPORTING_ENABLED=false
 ```
 
-## Using the CLI
+Find more about configuring Kapacitor [here](https://docs.influxdata.com/kapacitor/latest/introduction/installation/)
+
+### Exposed Ports
+
+- 9092 TCP -- HTTP API endpoint
```

Comment:
There needs to be a tab character between the `-` and `9092` rather than a space.